### ISSUE TITLE
Can't launch game anymore

### ISSUE THREAD

**Issue by ghost** [UnderscoreKilburn/repentance-issues#1813]
After the recent patch, when I try to launch Isaac I get an error message and the game closes itself. (Playing on steam version, Windows 7).

**Comment by ZMBanshee** [comment]
We're aware that some people are having problems starting the game now. The update was just for the G FUEL daily today, but there were some back end changes that caused unforeseen consequences. Since the G FUEL daily won't exactly be relevant after today, it will be rolled back to the previous version tomorrow, which should resolve any new issues related to the patch.

**Comment by TontonSancho** [comment]
**As investigation**
Game crashes at launch if GamePad is connected to computer
*BLUETOOTH HID v5.05 Gamepad [Xbox Wireless Controller 987A14FFB7F3*

At first, I did not realize, but :
since a couple of day : Repentance is crashing at startup (showing the game frame then close). The game generates a core dump.

My today investigation was :
If I unplug my controller, then the game starts.
If I plug the controller while the game already running, the game instantly crash, into coredump.

Stuff I tried :
  * Disabling Steam Input
  * Enabling Steam Input (Option : Always use Steam Input)
  * Disabling option.ini ControllerHotplug=0 ... but Can't get any controller working at all for playing the game.
  * Removing "Mod Restart with controller"
  * Disabling All Mods

Nothing solve the crashes.

If I try to start the game (but crashing instantly) with a controller connected, info.log is not written at all.  (not even touching the modification date of the file)
If i let the game to start with controller, then making it to crash by plugging a controller, the game writes into log.txt
The end of this file may help devs ... (I Hope) .. this is the last line :
```
[...]
[INFO] - Using XInput to initalize new device
```

I can provide coredumps.

Regards

As ping back: [The related topic on steam community](https://steamcommunity.com/app/250900/discussions/0/3826410862734014776/)
